scipy-1.0/paper.tex

```diff
@@ -281,6 +281,26 @@ \subsection*{Data structures}
 \input{ckdtree}
 \textbf{Sparse matrices}
 
+\texttt{scipy.sparse} offers seven sparse matrix data structures. 
+The most important ones are CSR and CSC; these offer fast row or column 
```

Comment:
...are the row- and column-compressed formats (CSR and CSC, respectively). These offer fast major-axis indexing and...